tests/test_umbral.py

```diff
@@ -84,7 +113,7 @@ def test_cfrag_serialization():
     priv_key = pre.gen_priv()
     pub_key = pre.priv2pub(priv_key)
 
-    _, capsule = pre.encapsulate(pub_key)
+    _, capsule = pre._encapsulate(pub_key)
```

Comment:
I don't really like `_` as a name in tests - do you think it makes sense to just call this `_key_bytes`?